### ISSUE TITLE
Source_textarea not working properly

### ISSUE THREAD

**Issue by Proteuss** [Frontwise/grid-editor#66]
1: When using textarea as input, the top toolbar to add rows doesn't appear.
First image is the regular editor. Second image is when i load the data from a textarea. The data loads properly but there is no top toolbar.
![image](https://user-images.githubusercontent.com/7173090/34675026-ec603ee4-f466-11e7-95a4-a6a3bf9891d1.png)

![image](https://user-images.githubusercontent.com/7173090/34675100-3cc425b2-f467-11e7-8ee7-28587eac1e7d.png)

2: I don't know if this is working as supposed. When i post the data i have to manually get the html from the editor and paste it on the textarea. By reading the examples, i thought the editor also used the textarea as output.

**Comment by Epskampie** [comment]
2. That's not how it's supposed to work. Someone else added the textarea support in a PR, so I'll have to look into it later.

Let me know if you find the fix yourself!

**Comment by Proteuss** [comment]
I managed to fix the toolbar. I commented this line on jquery.grideditor.js.
![image](https://user-images.githubusercontent.com/7173090/35052626-d27621ca-fb86-11e7-9634-617f5a7cbacf.png)
I don't know what that line does, but i think it is to use the textarea as output, because nothing broke when i deleted the line. 
Thank you for the editor, it's the best solution i've found to let users create html content.
